src/main/java/gov/cms/madie/services/TranslationServiceClient.java

```diff
@@ -41,4 +42,26 @@ public List<TranslatedLibrary> getTranslatedLibraries(String cql, String accessT
       throw new TranslationServiceException(msg, ex);
     }
   }
+
+  public String getMeasureBundleExport(Measure measure, String accessToken) {
```

Comment:
The name is a bit misleading. could be `getHumanReadable`?

Reply:
@adongare Just changed the name, thanks so much Ashok!

Reply:
Sure, np